app/serializers/concerns/user_sidebar_mixin.rb

```diff
@@ -44,6 +47,10 @@ def include_sidebar_list_destination?
     sidebar_navigation_menu?
   end
 
+  def sidebar_sections
+    object.sidebar_sections.order(created_at: :asc)
+  end
```

Comment:
I think this will conflict with `has_many :sidebar_sections` above. Not sure which is overriding which right now.